static/app/views/replays/detail/console/index.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from '@emotion/styled';
+
+import {Panel} from 'sentry/components/panels';
+import {BreadcrumbTypeDefault} from 'sentry/types/breadcrumbs';
+
+import ConsoleMessage from './consoleMessage';
+
+interface Props {
+  breadcrumbs: BreadcrumbTypeDefault[];
+  className?: string;
+}
+
+function BaseConsole({className, breadcrumbs}: Props) {
+  return (
+    <ConsoleTable className={className}>
+      {breadcrumbs.map((breadcrumb, i) => {
+        return (
+          <ConsoleMessage
+            key={i}
+            isLast={i === breadcrumbs.length - 1}
+            breadcrumb={breadcrumb}
+          />
+        );
+      })}
+    </ConsoleTable>
+  );
+}
+
+const Console = styled(BaseConsole)`
+  font-family: ${p => p.theme.text.familyMono};
+  font-size: 0.8em;
+`;
+
+const ConsoleTable = styled(Panel)`
+  display: grid;
+  grid-template-columns: max-content auto;
+`;
```

Comment:
we should be able to drop `className?: string` from the props if we combine the css here. i think all the css is being applied to the Panel now already. 

```
const ConsoleTable = styled(Panel)`
  font-family: ${p => p.theme.text.familyMono};
  font-size: 0.8em;
  display: grid;
  grid-template-columns: max-content auto;
`;
```

Reply:
Oops yeah I forgot to clean-up a bunch of stuff, thanks!